wasm/tx_ibc/src/lib.rs

```diff
@@ -7,13 +7,9 @@ use namada_tx_prelude::*;
 
 #[transaction]
 fn apply_tx(ctx: &mut Ctx, tx_data: BatchedTx) -> TxResult {
-    // let data = ctx.get_tx_data(&tx_data)?;
-
-    // let transfer =
-    // ibc::ibc_actions(ctx).execute(&data).into_storage_result()?;
-
-    // Temp. workaround for <https://github.com/anoma/namada/issues/1831>
-    let transfer = tx_ibc_execute()?;
```

Comment:
Since we are removing the host function can we also remove `crates/benches/txs.rs` and the `allowed_txs` bench entry in `crates/benches/Cargo.toml`? Given that everything runs in wasm now we don't need the benchmarks anymore

Reply:
done in the last 2 commits